src/objective-c/ProtoRPC/ProtoRPC.h

```diff
@@ -97,8 +97,36 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * GRPCProtoCallFlowControllable defines the available interface methods to use when flow control
+ * is enabled. Calling these methods when flow control is disabled has no effect.
+ */
+@protocol GRPCProtoCallFlowControllable <NSObject>
+
+/**
```

Comment:
Are these APIs for users? If so, probably provide them as public methods instead of protocol method?

Reply:
yes, these are APIs for users to call on.  This one wraps two receiveMessage methods in a single protocol so we can share the definition among GRPCUnaryProtoCall and GRPCStreamingProtoCall.  

Not sure what you mean by public methods though :  ) Since this protocol is defined in ProtoRPC.h it would already be public visible to callers whoever includes it.   Maybe I missed, or do we have some separate mechanism to mark a method public at the gRPC library level at release steps ?  LMK thanks !

Reply:
A little bit weird to wrap APIs in a protocol. Can we just create a base class with these APIs and let `GRPCUnaryProtoCall` and `GRPCStreamingProtoCall` to inhere from this class?

Reply:
I see, gotcha. Tried a little bit w/ the base class approach and there are some extra handling there to make sure init/new is not available publicly.   I think to keep it simple and not touching this interface too much, I got rid of the protocol and added these two methods directly to  GRPCUnaryProtoCall. Let me know if this works for u ;  )  Given that we already have similar methods on both interfaces (start / cancel / finish) this might be a good refactoring at some later point.   LMK, thanks !